src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/ScreenRecorder.cs

```diff
@@ -768,6 +768,20 @@ private IEnumerator RecordFrame(bool endRecording = false, bool endRecordingFrom
                             eventSystemInputModules = esGameObject.gameObject.GetComponents<BaseInputModule>().Where(a => a.isActiveAndEnabled).Select(a => a.GetType().FullName).ToList();
                         }
 
+                        var mainCamera = Camera.main;
```

Comment:
Oddly enough I think this can be null, I remember running into that issue in the past, so might want to do a null check here. https://docs.unity3d.com/ScriptReference/Camera-main.html#:~:text=If%20there%20is%20no%20enabled,valid%20result%20from%20its%20cache.

Reply:
Updated many places in the project to compensate for this